ui/redux/selectors/settings.js

```diff
@@ -121,8 +121,9 @@ export const selectWildWestDisabled = (state) => {
 
 export const selectosNotificationsEnabled = (state) => selectClientSetting(state, SETTINGS.OS_NOTIFICATIONS_ENABLED);
 
+export const selectDefaultChannelId = (state) => selectClientSetting(state, SETTINGS.ACTIVE_CHANNEL_CLAIM);
 export const selectDefaultChannelClaim = createSelector(
-  (state) => selectClaimForId(state, selectClientSetting(state, SETTINGS.ACTIVE_CHANNEL_CLAIM)),
+  (state) => selectClaimForId(state, selectDefaultChannelId(state)),
   (defaultChannelClaim) => defaultChannelClaim
 );
```

Comment:
_Can ignore if no time since this just pedantics, but avoid in the future or come back later to clean up_

<img width="526" alt="image" src="https://user-images.githubusercontent.com/64950861/190939355-d887ceb2-a9cb-42dd-b3cd-45364ac994b2.png">

The correction would probably be:
- Use `(selectClaimsById, selectDefaultChannelId)` as inputs instead
- Or don't cache at all since there is no transformation.